app/views/Mails/acceptrefuse/sendProposalApproved.scala.txt

```diff
@@ -34,53 +36,56 @@ Merci de ne pas répondre à cet email, mais d'utiliser votre interface pour éc
 Vous n'avez pas à vous inscrire, votre badge Speaker sera préparé pour vous à l'entrée de la conférence. Ce badge est valable pour les 3 jours de la conférence.
-Le diner speaker aura lieu le mercredi soir dans le centre de Paris. Nous vous tiendrons informé par email.
+Le diner speaker aura lieu le mercredi soir. Nous vous tiendrons informé par email quelques jours avant l'évènement.
 }
 
-Comment poser une question ?
-----------------------------------------------------------------------------------------------------------------------
-Merci de ne pas répondre à cet email, mais d'utiliser votre interface pour échanger avec le comité.
-
-@ConferenceDescriptor.current().conferenceUrls.cfpURL@routes.CallForPaper.showCommentForProposal(proposal.id)
-
-L'équipe de sélection de la conférence @Messages("longYearlyName")
+L'équipe @Messages("longYearlyName")
 
 ----------------------------------------------------------------------------------------------------------------------
 }
- Hi,
 
- We're glad to inform you that your proposal for @Messages("longYearlyName") has been accepted !
- Proposition title : @proposal.title, of type [@Messages(proposal.talkType.id)].
+Hi,
+
+We're glad to inform you that your proposal @proposal.title (@Messages(proposal.talkType.id)) has been accepted for @Messages("longYearlyName")!
+
+How to validate or not your participation?
+---------------------
+Note that you only have a few days to accept the terms and conditions and confirm your presence at @Messages("longYearlyName").
 
- Important information
- ---------------------
- You must now validate or decline the general terms and conditions, and confirm that you are still ok to do
- this presentation :
+Please validate now by following this link in your browser:
 
   @ConferenceDescriptor.current().conferenceUrls.cfpURL@routes.ApproveOrRefuse.showAcceptTerms()
 
+You will then have to accept or refuse your presentation(s).
+
+ A doubt? A question?
+ ------------------------------------
+ If you can’t confirm quickly, if you have any questions, please come back to us as soon as possible.
+
+ Do not reply to this email, but use the interface to exchange with the committee:
+
+  @ConferenceDescriptor.current().conferenceUrls.cfpURL@routes.CallForPaper.showCommentForProposal(proposal.id)
+
+
 @if(!ConferenceDescriptor.ConferenceProposalConfigurations.doesItGivesSpeakerFreeEntrance(proposal.talkType)) {
- This type of talk does not grant a conference pass, so you will need to register as an attendee in order to access the conference area.
+Attention, this presentation format for @Messages("longYearlyName") does not give a speaker badge. You must buy a seat at the conference to participate, if you do not have a ticket.
+
+The conference being full, you can still register via the following unique link (and linked to your proposal):
```

Comment:
```suggestion
Please note that this presentation format for @Messages("longYearlyName") does not provide a Speaker badge. You will need to purchase a seat at the conference to attend, if you do not have a ticket.

Since the conference is full, you can still register using the following unique link (linked to your proposal):
```